packages/react/src/components/select/select.stories.tsx

```diff
@@ -13,6 +13,8 @@ import { Meta, omit, reactMatrix, Story } from '../../../../../docs';
 const borderless = [true, false] as const;
 const disabled = [true, false] as const;
 const invalid = [true, false] as const;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const native = [true, false] as any;
```

Comment:
@rabelloo feel free to suggest an alternative here, I could not find a solution to this type issue.

Reply:
`as const` doesn't work?

Reply:
No, it does not.

That one is a little different, it has `true` described on native component props, and `false` on custom one.

Reply:
`as boolean[]` doesn't work either?
What's the expected type?

Reply:
Tried that, tried different variations - nothing worked.

It says type is `readonly true[] | readonly false[]` (or something like this), however if such set then there is another error...

Reply:
It's probably an issue in `reactMatrix`'s type, I can take a look later, `as any` is fine for now.
`as any[]` should probably work too.